backend/api/quivr_api/modules/knowledge/entity/knowledge_brain.py

```diff
@@ -27,5 +27,9 @@ class KnowledgeBrain(SQLModel, table=True):
     brain_id: UUID = Field(nullable=False, foreign_key="brains.brain_id")
     knowledge_id: UUID = Field(nullable=False, foreign_key="knowledge.id")
 
+    knowledge: "KnowledgeDB" = Relationship(back_populates="knowledge_brain")
```

Comment:
That's not how many-to-many model work in sqlmodel. Take a look at :

https://sqlmodel.tiangolo.com/tutorial/many-to-many/create-data/